apps/src/code-studio/components/SmallFooter.jsx

```diff
@@ -83,33 +72,6 @@ export default class SmallFooter extends React.Component {
     });
   };
 
-  minimizeOnClickAnywhere(event) {
-    // The first time we click anywhere, hide any open children
-    $(document.body).one(
-      'click',
-      function (event) {
-        // menu copyright has its own click handler
-        if (event.target === this.refs.menuCopyright) {
-          return;
-        }
-
-        this.setState({
-          menuState: MenuState.MINIMIZING,
-          moreOffset: 0,
-        });
-
-        // Create a window during which we can't show again, so that clicking
-        // on copyright doesnt immediately hide/reshow
-        setTimeout(
-          function () {
-            this.setState({menuState: MenuState.MINIMIZED});
-          }.bind(this),
-          200
-        );
-      }.bind(this)
-    );
-  }
-
```

Comment:
the new CopyrightDialog closes when the backdrop is clicked therefore we don't need this duplicated logic. In fact, it was causing the dialog to close when clicked.